Extension/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # C/C++ for Visual Studio Code Changelog
 
+## Version 1.17.5: August 28, 2023
+### Bug Fixes
+* Fix language server crash for platforms that don't support IntelliSense cache (AutoPCH). [#10789](https://github.com/microsoft/vscode-cpptools/issues/10789)
```

Comment:
Fix language server crash->Fix a language server crash 
don't support IntelliSense cache->don't support the IntelliSense cache